src/Tests/VideoBundleTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\media\Tests;
+
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Ensures that media bundle for videos can be created.
+ *
+ * @group media
+ */
+class VideoBundleTest extends WebTestBase {
```

Comment:
I'll fix these and the ones in the existing files and I'll make sure to pass all the future commits through this.